qa/qa_instance.py

```diff
@@ -606,15 +606,15 @@ def _TestKVMHotplug(instance, instance_info):
 
   """
   args_to_try = [
-    ["--net", "-1:add", "--hotplug"],
-    ["--net", "-1:modify,mac=aa:bb:cc:dd:ee:ff", "--hotplug", "--force"],
-    ["--net", "-1:remove", "--hotplug"]
+    ["--net", "-1:add"],
+    ["--net", "-1:modify,mac=aa:bb:cc:dd:ee:ff", "--force"],
+    ["--net", "-1:remove"]
   ]
```

Comment:
removing the option `--hotplug` here is good for tests in `_TestKVMHotplug(instance, instance_info)`. However in `TestInstanceModifyDisks(instance)` `--no-hotplug` should be set where disks are added, to represent/test the case without hot plugging.

Reply:
There are also `gnt-instance modify` operations for disk/net in `TestInstanceDeviceNames()`, these also need to explicitly disable hotplugging now.